### ISSUE TITLE
GameDLL-Autodetection

### ISSUE THREAD

**Issue by incognico** [jonnyboy0719/MetaMod-P#1]
SvenDS is started with -dll addons/metamod/dlls/metamod.so and no metamod config.ini is in place.

It creates meta_error.log:
GameDLL-Autodetection: Couldn't find gamedll in '/home/srcds/sc5/svencoop/dlls'.

Metamod works though, so it found the dll just fine.

**Comment by jonnyboy0719** [comment]
Are you using 'Unstable' version? I also want to note that SC 5.x metamod support might be abandoned since it will get replaced with SC5 own script system.

**Comment by incognico** [comment]
I'm using the release you have posted on the steam forums, Linux version.
As for Angelscript, I know, though it takes time to port all the
functionality/scripts over (and the lack of SQLite and Socket support is
forcing me to use amxx until that stuff is available in AS).

**Comment by jonnyboy0719** [comment]
yeah, they will probably get SQLite & Socket support soon enough, since people really want it (so we can replace MetaMod which gets out of date pretty quick, same with the engine binaries for SC 5.x)

if you are using this version of the metamod-p?
https://github.com/jonnyboy0719/MetaMod-P/releases/download/2.0.0/metamod.so

then it should work, if it keeps saying those thing and still work, then it might be a bug w/ metamod. Doesn't happen on Windows tho.
